src/index.tsx

```diff
@@ -674,27 +674,37 @@ const StyledEditor = styled("div")<{
     position: relative;
   }
 
-  img {
-    max-width: 100%;
-  }
-
   .image {
     text-align: center;
-
+    max-width: 100%;
     img {
       pointer-events: ${props => (props.readOnly ? "initial" : "none")};
+      display: inline-block;
+      max-width: 100%;
+      max-height: 75vh;
     }
   }
 
   .image.placeholder {
     position: relative;
     background: ${props => props.theme.background};
-
     img {
       opacity: 0.5;
     }
   }
 
+  .image-half-right {
+    float: right;
```

Comment:
I'm pretty sure it's totally fine to use float, image layout was really the primary usecase for this rule.